tests/unit/AmalgamationReviewConfirm.spec.ts

```diff
@@ -125,4 +129,90 @@ for (const test of amalgamationRegularBusinessInfo) {
       expect(section.findComponent(StaffPayment).exists()).toBe(true)
     })
   })
+
```

Comment:
Added a few tests for Short amalgamation (it is a bit of a different case than other filings)